packages/phone-number-privacy/src/match-making/get-contact-matches.ts

```diff
@@ -48,12 +49,15 @@ function isValidGetContactMatchesInput(requestBody: any): boolean {
   return (
     hasValidAccountParam(requestBody) &&
     hasValidUserPhoneNumberParam(requestBody) &&
-    hasValidContractPhoneNumbersParam(requestBody)
+    hasValidContractPhoneNumbersParam(requestBody) &&
+    isBodyReasonablySized(requestBody)
   )
 }
 
 function hasValidAccountParam(requestBody: any): boolean {
-  return requestBody.account && isValidAddress(requestBody.account)
+  return (
+    requestBody.account && isValidAddress(requestBody.account) && isBodyReasonablySized(requestBody)
```

Comment:
Why check `isBodyReasonablySized(requestBody)` here when we also check it above?

Reply:
Thanks for noticing this. I've moved these checks a new file, common/input-validation.ts.